pyproject.toml

```diff
@@ -79,10 +86,17 @@ dev = [
     "baybe[mypy]",
     "baybe[onnx]",
     "baybe[simulation]",
-    "baybe[telemetry]",
     "baybe[test]",
     "pip-audit>=2.5.5",
     "tox>=4.10.0",
+    # Following packages are necessary for telemetry. Due to issue with poetry
```

Comment:
I think you can actually remove them here. Having the `telemetry` group in `dev` was actually not needed (probably we added it just for better overview) but since it's contained in the core deps, you can skip them here

Reply:
Thx